cmd/katapult/data_centers.go

```diff
@@ -15,28 +17,29 @@ type dataCentersClient interface {
 	Get(ctx context.Context, ref core.DataCenterRef) (*core.DataCenter, *katapult.Response, error)
 }
 
+//go:embed formatdata/dcs/list.txt
+var dataCentersFormat string
+
+//go:embed formatdata/dcs/get.txt
+var getDataCenterFormat string
```

Comment:
Feel free to disagree, but I think it might be clearer and easier to just have the templates defined as literal strings in Go, as it makes it easier to get an overview of things by looking at a single file. Specially as the templates are very short of 1-3 lines of text only.